runtime.js

```diff
@@ -15,10 +17,23 @@ function checkProcessArgv (moduleName) {
 let preloadModules
 function checkPreloadModules (moduleName) {
   /* c8 ignore start */
-  // nullish needed for non Node.js runtime
+  // coverage - nullish needed for non Node.js runtime
   preloadModules ??= (process._preload_modules ?? [])
+
+  // coverage - TS specific
+  if (preloadModules.includes(moduleName)) {
+    return true
+  }
+
+  const modulePath = path.join(process.cwd(), 'node_modules', moduleName)
+  if (Object.keys(require.cache).some(k => k.startsWith(modulePath))) {
```

Comment:
The convention of this repo is using parentheses even with only one parameter, and `k` can be something more readable name IMO.